node/full_node_integration_test.go

```diff
@@ -353,6 +353,130 @@ func testSingleAggreatorSingleFullNodeSingleLightNode(t *testing.T) {
 	assert.Equal(n1h, n3h, "heights must match")
 }
 
+func checkAppCalls(assert *assert.Assertions, app *mocks.Application, expected []int) {
+	generateFraudProofCnt := 0
+	verifyFraudProofCnt := 0
+	for _, call := range app.Calls {
+		switch call.Method {
+		case "GenerateFraudProof":
+			generateFraudProofCnt++
+		case "VerifyFraudProof":
+			verifyFraudProofCnt++
+		}
```

Comment:
we should add a default case that panics to catch dev errors in test.

Reply:
we can't add default case as there exists other calls too like begin/end block. intention here is to only count the relevant calls and check